### DISCUSSION TITLE
[duplicate] "Configure Archive Logs" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#12593] [General]
# en/latest/configure-archive-logs/

https://argo-workflows.readthedocs.io/en/latest/configure-archive-logs/

**Comment by sfc-gh-rpeng** [comment]
Hi Argo team,
Any idea why it is not suggested to rely on Argo Workflows to archive logs? Archiving logs enables users to browse logs directly from the Argo UI as long as the workflow is not GCed.

**Reply by agilgur5** [reply]
Conventional logging utilities are purpose-built for storing, indexing, and searching logs. Artifact storage is a pretty naive way of doing it in comparison, and non-trivial to search etc.

**Reply by agilgur5** [reply]
The [links feature](https://argo-workflows.readthedocs.io/en/latest/links/) is one common way of linking to your logging facility from the Argo UI

**Reply by agilgur5** [reply]
More docs have been added to elaborate on this: #12597

**Comment by agilgur5** [comment]
Duplicate giscus discussion of #9970
